game_runner.rb

```diff
@@ -0,0 +1,23 @@
+class GameRunner
+  BOARD_TOP = " __ __ __\n"
+
+  def self.run(*args)
+    GameRunner.new.run(*args)
+  end
+
+  def run
+    puts empty_board(3)
+  end
+
+  def empty_board(rows)
```

Comment:
I was encouraged to follow a strict version of the `A method can't be more than 5 lines long` rule and I encourage you to do the same, with blank lines counting towards that total. It'll likely lead to thinking about the design differently and will establish those patterns from the get go. And probably be challenging!

You could remove the blank lines and meet the letter of the law, and I probably wouldn't stop you, but I'd maybe silently judge :)